packages/devtools_app/lib/src/ui/vm_flag_widgets.dart

```diff
@@ -85,11 +88,11 @@ class ProfileGranularityDropdown extends StatelessWidget {
     );
   }
 
-  Future<void> _onProfileGranularityChanged(String newValue) async {
+  Future<void> _onProfileGranularityChanged(String? newValue) async {
```

Comment:
can this be non-nullable?

Reply:
nope
This method is assigned to RoundedDropDownButton<String>.onChanged that has nullable parameter